canonical-websites/tutorials.ubuntu.com
# 147 update icons

## Done

- Update feedback icons
- Add github pull request template

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8016/](http://0.0.0.0:8016/)
- Go to the last page of a tutorial and see that the icons match those supplied in the issue


## Issue / Card

Fixes [#147](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/147)

## Screenshots

![screenshot from 2018-06-05 14-16-37](https://user-images.githubusercontent.com/501889/40978284-1b2c4de8-68cb-11e8-882d-7f7fa8b26260.png)
